src/gleam_stdlib.erl

```diff
@@ -411,6 +426,38 @@ inspect(Any) when is_function(Any) ->
 inspect(Any) ->
     ["//erl(", io_lib:format("~p", [Any]), ")"].
 
+inspect_maybe_gleam_atom([], none, []) ->
+    {error, cannot_be_an_empty_string};
+inspect_maybe_gleam_atom([Head | _Rest], none, []) when ?is_digit_char(Head) ->
+    {error, cannot_start_with_a_digit};
+inspect_maybe_gleam_atom(["_" | _Rest], none, []) ->
```

Comment:
I think you'll want the character `$_` rather than a string of one character here
```suggestion
inspect_maybe_gleam_atom([$_ | _Rest], none, []) ->
```